src/main/java/com/airlines/model/CrewMember.java

```diff
@@ -0,0 +1,71 @@
+package com.airlines.model;
+
+import java.time.LocalDate;
+
+public class CrewMember {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String position;
+    private LocalDate birthday;
+    private String citizenship;
```

Comment:
In term of OOP design, when you know there are fixed amount of values you should always consider using Enum, because it's add more safety for code using:
if (position.equals("PILO")) {
  ...
  
  if (citizenship.equals("YSA") {
  
  ...
  and so on.
  When there are some limited and rarely modificated set of predefined values, it's means we should use enum in order to prevent mistakes:
  
  if (citizenship == USA) {
   ...
   if (position == PILOT) {
    ...
    
compiler just won't let us use wrong values

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1269dee2e4effc40beaddfca6d85b7799518548f